docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,202 @@
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need access to some Dagster code.
+
+You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a **Dagster Cloud organization.** When your organization is created, a single instance deployment, named `prod`, will also be created. Instance deployments are similar to environments in that they allow you to operate independent instances of Dagster, with separately managed permissions.
+
+---
+
+## Step 2: Choose your deployment type
+
+In this step, you'll choose how your code is hosted and executed:
+
+- [Hybrid deployment](#hybrid-deployment)
+- [Serverless deployment](#serverless-deployment)
+
+### Hybrid deployment
+
+With a hybrid deployment, your Dagster code is executed in your infrastructure.
+
+Using a hybrid deployment requires setting up an agent for use with Dagster Cloud. The agent runs in your infrastructure, executes your code, and streams metadata about code execution over HTTPS back to Dagster Cloud.
+
+Dagster Cloud currently supports the following agents:
+
+- [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+- [Local](/dagster-cloud/deployment/agents/local)
+- [Docker](/dagster-cloud/deployment/agents/docker)
+- [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+
+**To use this deployment type**, follow the steps for setting up and deploying your agent of choice. Continue to the next step when finished.
+
+### Serverless deployment
+
+<Note>
+  Serverless deployment is currently in <strong>early access</strong>.
+</Note>
+
+Serverless deployment is a fully-managed version of Dagster Cloud that allows you to run your Dagster jobs without spinning up any infrastructure. While Serverless is the easiest way to get started with Dagster, this deployment type works best with jobs that aren't computationally expensive. Refer to the [Serverless guide](/dagster-cloud/deployment/serverless) for more info.
+
+If you find that Serverless isn't right for you, you can switch to a hybrid deployment at any time.
+
+**To use this deployment type**, click **Select** and confirm when prompted. **Note**: It may take a few minutes to complete the setup.
+
+---
+
+## Step 3: Deploy your code
+
+Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
```

Comment:
This step is a bit tricky. It might be nice if we could split the page entirely to serverless vs hybrid since the code deploying process is pretty different between the two.

For Serverless, I believe the only code deployment options we support are:
- Deploying using CI/CD, e.g. via GitHub Actions
- Deploying using the `dagster-cloud serverless` CLI
cc @prha  to make sure this is right

For hybrid, we support:
- Deploying using CI/CD, e.g via GitHub Actions
- Deploying using the `dagster-cloud workspace` CLI
- Deploying using the Workspace page

In an ideal world I think it would be great if we had a switcher like this to toggle the whole page between Serverless/Hybrid, but that's not a short-term priority 😄 

![Screen Shot 2022-08-04 at 11 19 48 AM](https://user-images.githubusercontent.com/10215173/182922608-3bf7f830-d63b-49b3-b8a8-bc00e64497ff.png)